keyboards/1upkeyboards/1up60rgb/keymaps/badger/new_tsangan.c

```diff
@@ -0,0 +1,59 @@
+/*
```

Comment:
This file actually can't be compiled due to the file name (keymap files must be named either `keymap.c` or `keymap.json`).

You could move this to `keyboards/1upkeyboards/1up60rgb/keymaps/badger-tsangan/keymap.c`, and then add a `keyboards/1upkeyboards/1up60rgb/keymaps/badger-tsangan/rules.mk` that contains `USER_NAME := badger`, and then compile either keymap as desired. (I'm thinking you have two of this board - one Tsangan bottom row and one standard, yes?)

Reply:
I addressed this by making the current keymap the default.  

I did order a second one of these boards, but it's been delayed for a while due to the current conditions.  Currently I've been testing my layouts by resoldering the bottom row :)

Reply:
> Lately I've been testing my layouts by resoldering the bottom row :)

That strikes me as tedious, but you do you. :)